wheatley/bot.py

```diff
@@ -270,6 +270,10 @@ def tick(self) -> None:
             self._place = 0
             self.start_next_row()
 
+            current_stroke = Stroke.from_index(self._row_number)
```

Comment:
We're in the gap between rows here and just added one to the row number, so is this the `next_stroke`?

Reply:
Oh yes very good point!